src/parsers/perf/perfparser.cpp

```diff
@@ -819,18 +819,8 @@ struct PerfParserPrivate
         summaryResult.cpuArchitecture = QString::fromUtf8(features.arch);
         summaryResult.cpusOnline = features.nrCpusOnline;
         summaryResult.cpusAvailable = features.nrCpusAvailable;
-        summaryResult.cpuSiblingCores = QString::fromUtf8(features.siblingCores.join(", "));
-
-        QList<QByteArray> siblingThreads;
-        for (QByteArray thread : features.siblingThreads) {
-            QList<QByteArray> siblings = thread.split(',');
-            for (QByteArray sibling : siblings) {
-                siblingThreads.append(sibling);
-            }
-        }
-        std::sort(siblingThreads.begin(), siblingThreads.end());
-        summaryResult.cpuSiblingThreads = QString::fromUtf8(siblingThreads.join(", "));
-
+        summaryResult.cpuSiblingCores = QString::fromUtf8(features.siblingCores.join(", ")).prepend(QLatin1Char('[')).replace(QLatin1Literal(", "), QLatin1Literal("], [")).append(QLatin1Char(']'));
```

Comment:
use QLatin1String instead of QLatin1Literal please, the latter is Qt 4 stuff afaik